go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
-	sigs.k8s.io/release-sdk v0.9.3
+	sigs.k8s.io/release-sdk v0.9.4-0.20221020144501-0f5793938dbc
```

Comment:
we should do a sdk release asap

Reply:
Let’s wait for https://github.com/kubernetes-sigs/release-sdk/issues/124